app-chores/lib/song.js

```diff
@@ -0,0 +1,30 @@
+class Song {
+	constructor(titulo, interpretes, disco, genero, duracion, archivo) {
+		this.titulo			= titulo;
+		this.interpretes	= this.interpretes;
+		this.genero			= genero; 
+		this.disco			= disco;
```

Comment:
Insisto, ¿qué parte de la lógica de negocio usa el disco en el que está una canción?
Además, todo este código se ha cambiado con un commit 8ac9de3dc4d0f1e5fe27744060ad9af2422c1588 que no hace referencia a ningún issue y que cambia (porque añade espacios) líneas que no corresponden al mensaje de commit, haciendo difícil su evaluación.

Reply:
Como en #15 comentabamos que las canciones se tenían que poder distinguir, pensé que el disco era importante, ya que un artista puede tener distintas versiones de una canción en distintos discos

Reply:
Tendrás entonces que plantear el problema de representar los discos, ¿no? ¿Que diga si son singles o LPs? ¿O si la canción salió primero en streaming, en single, EP, EP versión disco o LP? ¿O comprobar que el LP es del mismo intérprete y no, digamos, una recopilación de artistas?
Y llegados a ese punto, ¿crees que un producto mínimamente viable que permita representar una canción necesita el disco?

Reply:
La verdad que para la lógica de negocio presentada, no sería necesario, lo cambio :ok_hand:

Reply:
De verdad, no se trata de que lo cambiéis o no. Se trata de que justifiquéis, en el mensaje de commit que es como se comenta, las elecciones que habéis tomado y por qué.